Add mention of `retext-equality` for new words (#2)
get-alex/.github: contributing.md

### Adding profanities and other words

If you have profanities, insensitive words, and/or any other additions to add
to our repository, you’ll need to make a PR to either [`cuss`][cuss] or
[`retext-equality`][equality].

> The words will automatically be added into [`retext-profanities`][profanities]
> and alex as well.

Reviewer: ```suggestion
The words will automatically be used in alex as well.
```

your suggestion made this sentence incorrect.
